src/epd_driver/include/epd_driver.h

```diff
@@ -224,6 +243,16 @@ EpdRect epd_full_screen();
 void epd_copy_to_framebuffer(EpdRect image_area, const uint8_t *image_data,
                              uint8_t *framebuffer);
 
+/**
```

Comment:
Same for these functions. They are not public, so they should not appear in the public header.